src/components/StablecoinsTable.tsx

```diff
@@ -90,30 +88,26 @@ const StablecoinsTable = ({
           <Tr
             key={idx}
             color="text"
-            borderBottom={`1px solid ${textColor}`}
             _hover={{
               textDecoration: "none",
-              borderBottom: `1px solid ${textColor}`,
-              bg: "primary200",
-              color: "black300",
+              bg: "background.highlight"
             }}
             _focus={{
-              borderBottom: `1px solid ${textColor}`,
-              bg: "primary200",
-              color: "black300",
+              bg: "background.base",
+              color: "body.base",
             }}
           >
             <Td>
-              <Flex align="center">
+              <Flex align="center" pt={2}>
                 {image && <Image src={image} alt="" me={4} boxSize={6} />}
                 <>{name}</>
               </Flex>
             </Td>
             <Td>
-              <Flex align="center">{marketCap}</Flex>
+              <Flex align="center" pt={2}>{marketCap}</Flex>
             </Td>
             <Td>
-              <Flex align="center">{stablecoinsType[type]}</Flex>
+              <Flex align="center" pt={2}>{stablecoinsType[type]}</Flex>
```

Comment:
Would need to double check with the actual Figma, but would advocate that we not rely on padding-top for positioning these. @nloureiro is this small adjustment desired?

If we're adding `pt="2"` to these, we should likely ditch that and add 1 to the y-padding of the parent `Td` which currently has `p="4"`... adding `py="5"` would do this, but again, using 5 is a bit odd, and we may want to consider simplifying.

Reply:
I used pt 2 here to try to horizontally center the content as much as possible, but please give me some time, I will come up with a better solution in the next submission.

Reply:
> Would need to double check with the actual Figma, but would advocate that we not rely on padding-top for positioning these. @nloureiro is this small adjustment desired?
> 
> If we're adding `pt="2"` to these, we should likely ditch that and add 1 to the y-padding of the parent `Td` which currently has `p="4"`... adding `py="5"` would do this, but again, using 5 is a bit odd, and we may want to consider simplifying.

I used pt 2 here to try to horizontally center the content as much as possible, but please give me some time, I will come up with a better solution in the next submission.

Reply:
use `verticalAlign="inherit"` ！